lighthouse-cli/test/smokehouse/error-expectations.js

```diff
@@ -24,10 +26,10 @@ module.exports = [
     artifacts: {
       PageLoadError: {code: 'PAGE_HUNG'},
       devtoolsLogs: {
-        'pageLoadError-defaultPass': [/* ... */],
+        'pageLoadError-defaultPass': IS_ARRAY,
```

Comment:
I admit this is not ideal.

But "I expect an array with these exact values, and no more" is more common than "I expect an array", so I'd rather `[a, b, c]` mean the former and not "actual value is a subarray of expected array".

Reply:
agreed the array use case is more important

definitely weird that `IS_ARRAY` is an object instead of an array though :)

maybe we can comment this one-time usage that we can't use an array and at least assert `{length: '>0'}` or something?